src/main/java/org/kiwiproject/champagne/resource/ReleaseWithStatus.java

```diff
@@ -0,0 +1,20 @@
+package org.kiwiproject.champagne.resource;
+
+import java.util.Map;
+
+import org.kiwiproject.champagne.core.manualdeployment.DeploymentTaskStatus;
+import org.kiwiproject.champagne.core.manualdeployment.Release;
+
+import lombok.Builder;
+import lombok.Value;
+import lombok.experimental.Delegate;
+
+@Value
+@Builder
+public class ReleaseWithStatus {
```

Comment:
Maybe add a javadoc here...I always forget why we have this one...and why it's not with the other core/model classes.